experimental/dds/tree2/src/test/shared-tree/fuzz/anchorStability.fuzz.spec.ts

```diff
@@ -50,30 +50,20 @@ describe("Fuzz - anchor stability", () => {
 			validateConsistency: () => {},
 		};
 
+		let anchors: Map<Anchor, [UpPath, Value]>;
 		const emitter = new TypedEventEmitter<DDSFuzzHarnessEvents>();
 		emitter.on("testStart", (initialState: AbortFuzzTestState) => {
-			const firstAnchor = getFirstAnchor(initialState.clients[0].channel);
-			initialState.firstAnchor = firstAnchor;
-			initialState.clients[0].channel.transaction.start();
+			const tree = initialState.clients[0].channel;
+			tree.transaction.start();
+			anchors = createAnchors(tree);
 		});
```

Comment:
Since we're no longer using the `firstAnchor`  field of `AbortFuzzTestState` to keep track of the anchors, should we just remove this interface and go back to using `FuzzTestState`?